staging/cert-manager-setup/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: cert-manager-setup
 home: https://github.com/mesosphere/charts
-version: 0.1.6
+version: 0.1.7
```

Comment:
Technically not really a change to the chart, but `ct` lint detects the new `ci/test-values.yaml` that the e2e test script uses as a change, so...